client/models/service-instance.js

```diff
@@ -2,15 +2,6 @@ var request = require('request');
 var util = require('util');
 
 module.exports = function(ServiceInstance) {
-  function applyPatch(target, fileData, callback) {
-    return this._appCommand({
-      cmd: 'patch',
-      target: target,
-      patch: fileData
-    }, callback);
-  }
-  ServiceInstance.prototype.applyPatch = applyPatch;
```

Comment:
This appears to be part of the JS api and not just the remote API, which would require a semver major bump. Is there a way not to remove this method?